hail/src/main/scala/is/hail/asm4s/CodeBuilder.scala

```diff
@@ -33,14 +33,14 @@ trait CodeBuilderLike {
 
   def isOpenEnded: Boolean
 
-  // def code: Code[Unit] // debugging only
+   def code: Code[Unit] // debugging only
```

Comment:
debug stuff here

Reply:
Beat me to it by a second, it's gone now.